src/main/java/com/mfahproj/webapp/Database.java

```diff
@@ -22,7 +18,7 @@ public static enum Result {
 
     // Connection requirements.
     private static String URL = "jdbc:mysql://localhost:3306/museum";
-    private static String USER = "placeholder";
+    private static String USER = "root";
```

Comment:
Do not need to update the default user from placeholder. Do not want to risk logging in to the root account by default. Placeholder should fail correctly if the they have not updated "app.java"